contracts/ZecaleDispatcher.sol

```diff
@@ -46,7 +46,7 @@ contract ZecaleDispatcher
         _vk = vk;
         // Compute expected inputs per batch
         _total_inputs = Groth16BW6_761.num_inputs_from_vk_length(vk.length);
-        _inputs_per_nested_tx = (_total_inputs - 1) / batch_size;
+        _inputs_per_nested_tx = (_total_inputs - 2) / batch_size;
```

Comment:
`-2` because the vk digest is taking 2 EVM words now, right?

Reply:
The vk digest change is independent (and actually doesn't affect this layout, since we always pass BW6-761 scalars).  The -2 is because we now have 2 scalars (vk_hash and results) in front of the nested_tx inputs.  Hopefully it's clear now that the layout is updated at the top of the file.

Reply:
Ah yes, that's all clearer, thanks.